### ISSUE TITLE
Will the v1 pcb fit the v2 case?

### ISSUE THREAD

**Issue by dhanakane** [coseyfannitutti/discipline#25]
A 3D printing shop has mentioned that the v1 pcb will not fit the v2 cases. It looks like the layouts are identical bar a couple of standoff holes. Is this accurate please?

**Comment by coseyfannitutti** [comment]
Yes, both versions of the PCB will fit in the acrylic case found in this
repository.

The reason for the P3D case not being compatible is the mounting style. The
P3D case uses the tray mount style mounting holes with standoffs. The
mounting holes were shifted slightly on the newer version of the PCB. This
acrylic case, however, does not use those mounting holes, so both versions
are compatible.
